src/cljs/rems/atoms.cljs

```diff
@@ -324,7 +324,7 @@
          :label (:label action)
          :href (:url action)
          :on-click (:on-click action)
-         :class (str "btn btn-link " (:class action))}])
+         :class (str "dropdown-item btn btn-link py-2 " (:class action))}])
```

Comment:
Maybe it would be better to have the `dropdown-item` class passed from the use of this component so that `action-link` itself could be used outside of dropdowns? So could be passed in the `:class`.

Reply:
Yeah, that is better for re-usability. I was initially unsure where this atom was used, and since `:class` property can receive either string or list, I went for "simpler" solution so as to not make a decision about it.